packages/components/src/ui/visually-hidden/use-visually-hidden.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { cx } from '@wp-g2/styles';
+
+/**
+ * Internal dependencies
+ */
+import * as styles from './visually-hidden-styles';
+
+/**
+ *
+ * @param {import('@wp-g2/create-styles').ViewOwnProps<{}, 'div'>} props
+ */
+export function useVisuallyHidden( { className, ...props } ) {
+	// circumvent the context system and write the classnames ourselves
+	const classes = cx(
+		'components-visually-hidden wp-components-visually-hidden',
```

Comment:
Out of curiosity, why does it have two similar class names?

The old implementation uses only `components-visually-hidden`.

Reply:
G2 introduces the `wp-` prefixed classname. Typically they'd be added by `useContextSystem` here: https://github.com/ItsJonQ/g2/blob/5feb4f2822efcd67afbf3f52849c78b8cbfd0f82/packages/context/src/use-context-system.js#L95

For consistency's sake I added both to the hooks version.